modules/charon-core/src/main/java/org/wso2/charon3/core/utils/ResourceManagerUtil.java

```diff
@@ -404,15 +405,16 @@ public static int processCount(String countStr) throws BadRequestException {
         } else {
             try {
                 count = Integer.parseInt(countStr);
+                if (count == 0) {
+                    return -1;
```

Comment:
why we explicitly return -1 when the count is 0?

Reply:
because we return all the users when the count value is zero. Therefore i have used -1 as a separate scenario to handle count=0 instance